locales/en.json

```diff
@@ -189,12 +189,17 @@
     "diagnosis_mail_outgoing_port_25_blocked_details": "You should first try to unblock it in your internet service provider (or hosting provider) configuration panel or by sending a ticket to your hosting provider. Meanwhile, the server won't be able to send emails to other servers.",
```

Comment:
=> "diagnosis_mail_outgoing_port_25_blocked": "Outgoing port 25 appears to be **bloecked** in IPv{ipversion}",